docs/data/data-grid/editing/CustomEditComponent.js

```diff
@@ -31,6 +31,10 @@ function RatingEditInputCell(props) {
     }
   };
```

Comment:
https://user-images.githubusercontent.com/13808724/178468467-5f8418e3-fb39-4ff1-81f8-a7efe878163b.mov

Demo throws an error when clicking "Save" second time

Reply:
Fixed in 4998ca80